apps/frontend-pwa/src/components/liveSession/QuestionArea.tsx

```diff
@@ -0,0 +1,311 @@
+import { push } from '@socialgouv/matomo-next'
+import { H1 } from '@uzh-bf/design-system'
+import dayjs from 'dayjs'
+import localForage from 'localforage'
+import _without from 'lodash/without'
+import React, { useEffect, useState } from 'react'
+import v8n from 'v8n'
+
+import { QUESTION_GROUPS, QUESTION_TYPES } from '../../constants'
+import FREETextAnswerOptions from '../questions/FREETextAnswerOptions'
+import NUMERICALAnswerOptions from '../questions/NUMERICALAnswerOptions'
+import QuestionDescription from '../questions/QuestionDescription'
+import SCAnswerOptions from '../questions/SCAnswerOptions'
+import SessionProgress from './SessionProgress'
+
+// TODO: notifications
+
+interface QuestionAreaProps {
+  expiresAt?: Date
+  questions: any[] // { content: string, contentPlain: string, id: string, name: string, type: string, options: any }[]
+  handleNewResponse: any // TODO: correct typing
+  sessionId: string
+  timeLimit?: number
+  isStaticPreview?: boolean
+}
+
+const messages = {
+  [QUESTION_TYPES.SC]: <p>Bitte eine einzige Option auswählen:</p>,
+  [QUESTION_TYPES.MC]: <p>Bitte eine oder mehrere Optionen auswählen:</p>,
+  [QUESTION_TYPES.FREE_TEXT]: <p>Bitte eine Antwort eingeben:</p>,
+  [QUESTION_TYPES.NUMERICAL]: (
+    <p>Bitte eine Antwort aus dem vorgegebenen Bereich auswählen:</p>
+  ),
+}
+
+function QuestionArea({
+  expiresAt,
+  questions,
+  handleNewResponse,
+  sessionId,
+  timeLimit,
+}: QuestionAreaProps): React.ReactElement {
+  const [remainingQuestions, setRemainingQuestions] = useState(new Array())
+  const [activeQuestion, setActiveQuestion] = useState(
+    (): any => remainingQuestions[0]
+  )
+  const currentQuestion = questions[activeQuestion]
+
+  const [{ inputValue, inputValid, inputEmpty }, setInputState] = useState({
+    inputEmpty: true,
+    inputValid: false,
+    inputValue: QUESTION_GROUPS.CHOICES.includes(
+      questions[remainingQuestions[0]]?.type
+    )
+      ? new Array(questions[remainingQuestions[0]]?.options.length, false)
+      : undefined,
+  })
+
+  // TODO: fix response storage, currently multiple replays are possible - maybe automatically resolved with handleNewResponse implementation
```

Comment:
this will only be needed when responding anonymously. if logged in, we can verify server-side that a participant does not respond twice.